README.md

```diff
@@ -185,12 +268,42 @@ and [development](https://docs.ansible.com/ansible/latest/dev_guide/index.html).
 * File headers and functions should have comments for their intent.
 
 ## Ansible Best Practices
-
+* Ansible variables use lazy evaluation. [more_info](https://github.com/ansible/ansible/issues/10374)
+* All tags should be namespaced/prefixed with the role name. 
+* Use the `command` module instead of the `shell` module when use of either is required.
+  In all other cases use a dedicated module, if it exists. If not, see the [section](#check-mode-and-idempotency-issues) 
+  about idempotency and check mode and make sure that you support them properly (your task
+  will likely need conditionals such as `changed_when:` and maybe `check_mode:` ). 
+  Anytime `command` or `shell` modules are used; a comment in the code with justificiation
+  would help with future maintenance.
+* Beware of bare variables in `when`. Use '`| bool`' if the variable is supposed to hold a boolean
+  value. [more_info](https://github.com/ansible/ansible/issues/39414).
+* Do not use `meta: end_play`. It aborts the whole play instead of a given host (with multiple
+  hosts in the inventory) [more_info](https://github.com/ansible/ansible/issues/27973) - We may
+  consider using `meta: end_host` but this was recently introduced in Ansible 2.8 [more_info](https://github.com/ansible/ansible/pull/47194)
+* Put variables in Jinja2 templates and if reasonable into task names, this helps with reading
```

Comment:
What do you mean by "in Jinja2 templates and if reasonable into task names"? The original document reads "Put the variables in Jinja2 templates if reasonable into task names" and that's what was intended. I.e. put variables into task names, but wrapped into Jinja2 templates (because otherwise they would not get expanded).
Maybe the original formulation is not exactly clear, but your reformulation does not make sense.

Reply:
I suggest this:

"If reasonable, task names can be made dynamic by using variables (wrapped in Jinja2 templates), this helps with reading the logs. On the other hand, don't do this for play names, variables don't get expanded properly there."